angular2 tests failing in master

As I write this master is at 3fc1377ce29dd9670975cb68615edf1b819fc4a5 from PR #6207.

If I run `npm run all` I see failures with angular2. I'm currently running node v4.1.2 on Mac OS.

```
----------------- For file:angular2/router-2.0.0-alpha.38.d.ts
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.38.d.ts(302,25): error TS2304: Cannot find name 'StringMap'.
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.38.d.ts(304,13): error TS2304: Cannot find name 'StringMap'.
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.38.d.ts(887,86): error TS2304: Cannot find name 'StringMap'.
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.38.d.ts(893,21): error TS2304: Cannot find name 'StringMap'.
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.38.d.ts(925,13): error TS2304: Cannot find name 'StringMap'.

----------------- For file:angular2/router-2.0.0-alpha.36.d.ts
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.36.d.ts(224,14): error TS2304: Cannot find name 'StringMap'.
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.36.d.ts(531,22): error TS2304: Cannot find name 'StringMap'.
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.36.d.ts(552,14): error TS2304: Cannot find name 'StringMap'.
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.36.d.ts(573,14): error TS2304: Cannot find name 'StringMap'.

----------------- For file:angular2/router-2.0.0-alpha.37.d.ts
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.37.d.ts(273,14): error TS2304: Cannot find name 'StringMap'.
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.37.d.ts(582,22): error TS2304: Cannot find name 'StringMap'.
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.37.d.ts(609,14): error TS2304: Cannot find name 'StringMap'.
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.37.d.ts(634,14): error TS2304: Cannot find name 'StringMap'.

----------------- For file:angular2/router-2.0.0-alpha.35.d.ts
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.35.d.ts(224,14): error TS2304: Cannot find name 'StringMap'.
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.35.d.ts(531,22): error TS2304: Cannot find name 'StringMap'.
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.35.d.ts(552,14): error TS2304: Cannot find name 'StringMap'.
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/router-2.0.0-alpha.35.d.ts(573,14): error TS2304: Cannot find name 'StringMap'.

----------------- For file:angular2/http-2.0.0-alpha.38.d.ts
/Users/jim/RedSeal/hub/DefinitelyTyped/angular2/http-2.0.0-alpha.38.d.ts(914,37): error TS2304: Cannot find name 'StringMap'.
```

--- a/angular2/http-2.0.0-alpha.37.d.ts
+++ b/angular2/http-2.0.0-alpha.37.d.ts
@@ -13,7 +13,7 @@
 // If you don't have them installed you can install them using TSD
 // https://github.com/DefinitelyTyped/tsd
 
-///<reference path="./angular2.d.ts"/>
+///<reference path="./angular2-2.0.0-alpha.37.d.ts"/>
 
 
 

--- a/angular2/http-2.0.0-alpha.38.d.ts
+++ b/angular2/http-2.0.0-alpha.38.d.ts
@@ -13,7 +13,7 @@
 // If you don't have them installed you can install them using TSD
 // https://github.com/DefinitelyTyped/tsd
 
-///<reference path="./angular2.d.ts"/>
+///<reference path="./angular2-2.0.0-alpha.38.d.ts"/>
 
 
 

--- a/angular2/http-2.0.0-alpha.39.d.ts
+++ b/angular2/http-2.0.0-alpha.39.d.ts
@@ -13,7 +13,7 @@
 // If you don't have them installed you can install them using TSD
 // https://github.com/DefinitelyTyped/tsd
 
-///<reference path="./angular2.d.ts"/>
+///<reference path="./angular2-2.0.0-alpha.39.d.ts"/>
 
 
 

--- a/angular2/router-2.0.0-alpha.31.d.ts
+++ b/angular2/router-2.0.0-alpha.31.d.ts
@@ -8,7 +8,7 @@
 // Please do not create manual edits or send pull requests
 // modifying this file.
 // ***********************************************************
-///<reference path="./angular2.d.ts"/>
+///<reference path="./angular2-2.0.0-alpha.31.d.ts"/>
 
 
 

--- a/angular2/router-2.0.0-alpha.35.d.ts
+++ b/angular2/router-2.0.0-alpha.35.d.ts
@@ -13,7 +13,7 @@
 // If you don't have them installed you can install them using TSD
 // https://github.com/DefinitelyTyped/tsd
 
-///<reference path="./angular2.d.ts"/>
+///<reference path="./angular2-2.0.0-alpha.35.d.ts"/>
 
 
 

--- a/angular2/router-2.0.0-alpha.36.d.ts
+++ b/angular2/router-2.0.0-alpha.36.d.ts
@@ -13,7 +13,7 @@
 // If you don't have them installed you can install them using TSD
 // https://github.com/DefinitelyTyped/tsd
 
-///<reference path="./angular2.d.ts"/>
+///<reference path="./angular2-2.0.0-alpha.36.d.ts"/>
 
 
 

--- a/angular2/router-2.0.0-alpha.37.d.ts
+++ b/angular2/router-2.0.0-alpha.37.d.ts
@@ -13,7 +13,7 @@
 // If you don't have them installed you can install them using TSD
 // https://github.com/DefinitelyTyped/tsd
 
-///<reference path="./angular2.d.ts"/>
+///<reference path="./angular2-2.0.0-alpha.37.d.ts"/>
 
 
 

--- a/angular2/router-2.0.0-alpha.38.d.ts
+++ b/angular2/router-2.0.0-alpha.38.d.ts
@@ -13,7 +13,7 @@
 // If you don't have them installed you can install them using TSD
 // https://github.com/DefinitelyTyped/tsd
 
-///<reference path="./angular2.d.ts"/>
+///<reference path="./angular2-2.0.0-alpha.38.d.ts"/>
 
 
 

--- a/angular2/router-2.0.0-alpha.39.d.ts
+++ b/angular2/router-2.0.0-alpha.39.d.ts
@@ -13,7 +13,7 @@
 // If you don't have them installed you can install them using TSD
 // https://github.com/DefinitelyTyped/tsd
 
-///<reference path="./angular2.d.ts"/>
+///<reference path="./angular2-2.0.0-alpha.39.d.ts"/>
 
 
 

--- a/angular2/test_lib-2.0.0-alpha.38.d.ts
+++ b/angular2/test_lib-2.0.0-alpha.38.d.ts
@@ -13,7 +13,7 @@
 // If you don't have them installed you can install them using TSD
 // https://github.com/DefinitelyTyped/tsd
 
-///<reference path="./angular2.d.ts"/>
+///<reference path="./angular2-2.0.0-alpha.38.d.ts"/>
 ///<reference path="../jasmine/jasmine.d.ts"/>
 
 

--- a/angular2/test_lib-2.0.0-alpha.39.d.ts
+++ b/angular2/test_lib-2.0.0-alpha.39.d.ts
@@ -13,7 +13,7 @@
 // If you don't have them installed you can install them using TSD
 // https://github.com/DefinitelyTyped/tsd
 
-///<reference path="./angular2.d.ts"/>
+///<reference path="./angular2-2.0.0-alpha.39.d.ts"/>
 ///<reference path="../jasmine/jasmine.d.ts"/>
 
 

